cmake/onnxruntime_common.cmake

```diff
@@ -135,3 +135,7 @@ if ("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
     target_link_libraries(onnxruntime_common rt)
   endif()
 endif()
+
+if (onnxruntime_WINML_NAMESPACE_OVERRIDE STREQUAL "Windows")
```

Comment:
>onnxruntime_WINML_NAMESPACE_OVERRIDE [](start = 4, length = 36)

It looks we don't override the namespace in WAI-nightly-build.

Reply:
Where can I find this "Windows" override?

Reply:
It is set here : https://microsoft.visualstudio.com/windowsai/_git/WindowsAI?path=%2FCMakeLists.txt&version=GBmaster&line=176&lineEnd=177&lineStartColumn=1&lineEndColumn=1&lineStyle=plain

Reply:
![image](https://user-images.githubusercontent.com/16694211/94179429-55973380-fe51-11ea-80e2-96f4d226a2fb.png)

Reply:
From my WAI build run : 
![image](https://user-images.githubusercontent.com/16694211/94182704-3d75e300-fe56-11ea-82c5-c883c0927dd3.png)